quodlibet/po/fr.po

```diff
@@ -523,9 +513,8 @@ msgid "Internet Radio"
 msgstr "Radio Internet"
 
 #: ../quodlibet/browsers/iradio.py:252
-#, fuzzy
 msgid "Downloading station list"
-msgstr "Télécharger les fichiers"
+msgstr "Télécharger la liste de station"
```

Comment:
Oh, strange, I've been seeing that written a numerous time when I was working in a council during 3 years in Australia.

Anyway, fix pushed.